content/docs/user-guide/gems/reference/ros2/_index.md

```diff
@@ -0,0 +1,119 @@
+
+<!-- ## Clone this repository
+
+Clone the gem repository from Github:
+
+```
+git clone https://github.com/RobotecAI/o3de-ros2-gem.git
+```
+
+## Building
+
+The Gem is built through building a project which uses it. Make sure to
+[source your ros2 workspace](https://docs.ros.org/en/rolling/Tutorials/Configuring-ROS2-Environment.html#source-the-setup-files)
+before building.
+
+## Adding Gem to your project
+
+To use this Gem in your project, you need to register the Gem with O3DE. First, clone this repository. Then,
+in o3de folder:
+```
+scripts/o3de.sh register --gem-path <PATH_TO_CLONED_ROS2_GEM>
+scripts/o3de.sh enable-gem -gn ROS2 -pp <PATH_TO_YOUR_PROJECT>
+```
+
+## Example project
+
+You can test O3DE ROS2 Gem with [this project](https://github.com/RobotecAI/Ros2WarehouseDemo).
+It will allow you to run robot navigation. All necessary assets are included. -->
+
+## User Guides
+
+Follow the [ROS 2 Gem User Guide](ros2-gem-user-guide.md) to understand its concepts and components.
+
+<!-- If you plan on contributing please follow the [Pull Request Publishing](docs/guides/pr_publishing.md) guide. For those using the Clion IDE we advise to follow the [Development in Clion](docs/guides/development_in_clion.md) guide. -->
+
+## How to create your own robotic simulation
+
+>This section is to be detailed.
+
+Once you are set up and familiar with the example project, consider the following steps:
+1. [Create a new O3DE project](/docs/welcome-guide/create/) 
+2. Register ROS2 Gem and other useful Gems to created project. 
+Follow [Registering Gems to a Project](/docs/user-guide/project-config/register-gems/) guide. 
+3. Create or import Assets for your robots and environment. 
+   1. You can use formats supported by O3DE.
+   2. You can import your robot from URDF.
+   3. Imported models might require some adjustments to be simulation-ready.
+4. Determine which sensors you need to simulate. 
+   1. Some sensors are already implemented in this Gem.
+      1. They might require specialization (implementation specific for particular models).
+      2. You might want to consider tradeoffs between performance and realism in each case.
+   2. Use ROS2SensorComponent as a base class if you are implementing a new sensor.
+5. Develop necessary sensors and their prefabs.
+6. Consider developing additional abstraction to handle spawning and despawning robots.
+   1. This would also be a valuable contribution to the Gem.
+7. Develop your scene and simulation scenario, placing Assets and configuring Components.
+
+Enjoy simulation with some of many [ROS2 packages](https://index.ros.org/packages/#humble) and projects in [ROS2 ecosystem](https://project-awesome.org/fkromer/awesome-ros2).
+
+## Acknowledgements
+
+This project was originally developed by [Robotec.ai](https://robotec.ai) in cooperation with [AWS Game Tech](https://aws.amazon.com/gametech/) and [AWS RoboMaker](https://aws.amazon.com/robomaker/).
```

Comment:
This no longer refers to the project itself, it is fine to leave Acknowledgements in the README only